python/pyspark/sql/catalog.py

```diff
@@ -613,47 +786,76 @@ def createTable(
             df = self._jcatalog.createTable(tableName, source, scala_datatype, description, options)
         return DataFrame(df, self._sparkSession)
 
-    def dropTempView(self, viewName: str) -> None:
+    def dropTempView(self, viewName: str) -> bool:
```

Comment:
Is this a API change or user behavior change?

Reply:
Nope, this is a bug fix. It just fixes the type hint which does not break something.